### DISCUSSION TITLE
CRLF crashed our renovate pipeline

### DISCUSSION THREAD

**Discussion by rhiannonhetty** [renovatebot/renovate#26255] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

the latest (12 dec 23) - it renovates itself

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps (dev.azure.com)

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

Despite renovate docs stating it supports windows, when using line separator CRLF for our gradle.properties, renovate gave us the error 'error updating branch' with no additional details. After looking at what the affected services had in common (not all our services were returning this error) we found that the affected services had their gradle.properties set as CRLF for new lines. When we changed to mac OS LF, all affected services were renovated successfully.

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
<img width="644" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/6f581dfd-04f1-43d4-b581-18a75d9ce790">

**Reply by rhiannonhetty** [reply]
Hi, thanks for sharing the solution! The main issue for us is that we had no idea what the error was, since the error in the logs was completely unrelated and so only noticed the line separator by chance. It might be good to have a custom error for this scenario as it's a difficult one to spot.
